asyncio_run_in_process/run_in_process.py

```diff
@@ -346,3 +355,14 @@ async def run_in_process(async_fn: Callable[..., TReturn],
     async with proc_ctx as proc:
         await proc.wait()
     return proc.get_result_or_raise()
+
+
+async def run_in_process_with_trio(async_fn: Callable[..., TReturn],
+                                   *args: Any,
+                                   loop: asyncio.AbstractEventLoop = None,
```

Comment:
Looks like we can drop this argument from the function signature?

Reply:
Oops, indeed, that was not meant to be there.